waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -73,6 +88,37 @@ deleteGeneratedMigrationsDirIfRedundant spec projectRootDir = do
   where
     projectMigrationsDirAbsFilePath = SP.fromAbsDir $ projectRootDir </> dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
 
+-- | This function warns the user to run `wasp db migrate-dev` under certain conditions. But first, a few preliminaries:
+--   - This function relies on the latest schema.prisma, and thus should run after the writing of FileDrafts has occurred.
+--   - A non-empty schema.prisma should always exist in the generated project dir, even if the user has no entities defined.
+--     `wasp new` will create one with a `datasource db {}` and `generator client {}`, for example.
+--   - A schema.prisma.wasp-checksum file is created/updated each time `wasp db migrate-dev` is run locally and contains checksum(schema.prisma).
```

Comment:
```suggestion
-- | Checks if user needs to run `wasp db migrate-dev` due to changes they did in
-- schema.prisma, and if so, returns a warning.
-- When doing this, it looks at schema.prisma in the generated project.
--
-- This function makes following assumptions:
--  - schema.prisma will exist in the generated project even if no Entities were defined.
--    Due to how Prisma itself works, this assumption is currently fulfilled.
--  - schema.prisma.wasp-checksum contains the checksum of the schema.prisma
--    as it was during the last `wasp db migrate-dev`.
```

Why this suggestion:
- Clarifies a bit what is the function doing vs what it is assuming.
- Relaxes last assumption to focus on what schema.prisma.wasp-checksum contains vs how it was generated (very similar though).

Btw I am aware that it is easy for me to propose changes once you write the content :D.

Reply:
No sweat :D, can update comment 👍🏻